WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleItemListEmptyView.swift

```diff
@@ -5,42 +5,75 @@ struct PointOfSaleItemListEmptyView: View {
     @Environment(\.floatingControlAreaSize) private var floatingControlAreaSize: CGSize
     private let baseItem: ItemListBaseItem
 
-    init(base: ItemListBaseItem) {
+    private let onAction: (() -> Void)?
+
+    @State private var viewWidth: CGFloat = 0
+
+    init(base: ItemListBaseItem, onAction: (() -> Void)? = nil) {
         self.baseItem = base
+        self.onAction = onAction
     }
 
     var body: some View {
-        VStack(alignment: .center, spacing: PointOfSaleItemListErrorLayout.headerSpacing) {
+        VStack {
             Spacer()
-            Image(decorative: PointOfSaleAssets.magnifierNotFound.imageName)
-                .resizable()
-                .aspectRatio(contentMode: .fit)
-                .frame(width: Constants.iconSize, height: Constants.iconSize)
-                .foregroundColor(.posOnSurfaceVariantHighest)
-                .renderedIf(!dynamicTypeSize.isAccessibilitySize)
-            Text(title)
-                .foregroundStyle(Color.posOnSurfaceVariantHighest)
-                .font(.posHeadingBold)
-            Text(subtitle)
-                .foregroundStyle(Color.posOnSurfaceVariantHighest)
-                .font(.posBodyLargeRegular())
-                .padding([.leading, .trailing])
-            Text(hint)
-                .foregroundStyle(Color.posOnSurfaceVariantHighest)
-                .font(.posBodyLargeRegular())
-                .padding([.leading, .trailing])
+            VStack(alignment: .center, spacing: POSSpacing.none) {
```

Comment:
I believe the changes here creates a regression in the empty state when the feature flag is off, I've dropped some images in the response 👍

Reply:
Thanks for catching this!